make/common/native/Link.gmk

```diff
@@ -198,4 +198,16 @@ define CreateDynamicLibraryOrExecutable
 	      $(CODESIGN) -f -s $$($1_CODESIGN_OPTS) --entitlements \
 	          $$(call GetEntitlementsFile, $$@) $$@)
         endif
+
+  # This is for IDE integration purposes only, and is not normally generated
+  $1_LD_JSON := $$(MAKESUPPORT_OUTPUTDIR)/compile-commands/$$($1_NAME)-ldflags.txt
```

Comment:
Should this be called `JSON` if it's not actually a json file? In the other file it's referenced as a LINKER_FLAGS_FILE.

Reply:
Good catch, thanks. Too much copy/paste.